### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by martinkeat** [harvester/harvester#2752]
**Describe the bug**
Installation hanging at "extracting file run.sh...."

System is a Dell R710 server with a raid 1 array (ssd) being used for the primary installation and a raid 0 array (ssd) for the VM drive.

Able to go through all of the options at the beginning of the install, and then it just hangs when getting to the above. Have tried both UEFI and BIOS options for boot. 

**To Reproduce**
1. Boot system with ISO
2. Select install options "new cluster", setup network (static ips) and dns et al. 
3. Commence install

**Expected behavior**
Should install but hanging instead. 

**Support bundle**

**Environment**
 - Harvester ISO version: latest 
 - Underlying Infrastructure : Bare metal Dell R710
 - 
![Capture](https://user-images.githubusercontent.com/8785775/189018203-0cd54161-3f6e-4b92-85d0-d6dff38383f8.JPG)

**Additional context**

**Comment by RegisWeduLabs** [comment]
@martinkeat, Is it installed directly on the server or you are going over WAN/LAN with a remote administration tool (IPMI/DRAC)?  If going over DRAC/IPMI, please have a look at https://github.com/harvester/harvester/issues/2651#issuecomment-1218389598

**Comment by irishgordo** [comment]
I agree with @RegisHubelia  - this seems to be related directly to #2651 

In 2651, there are a few workarounds @martinkeat  that you could try as an act to keep the node moving along when it reaches this state:
https://github.com/harvester/harvester/issues/2651#issuecomment-1218280683

**Comment by xeruf** [comment]
Same issue, guess this can be closed.
Should be fixed with 1.2

**Comment by mauriiciodias** [comment]
@irishgordo this bug it's resolved in 1.2.0-rc2?
I need to make a poc but I am facing this bug in the latest version.

**Comment by xeruf** [comment]
@mauriiciodias I have the same issue, see the linked issue, basically `curl` the iso to a local storage medium and install from there

**Comment by irishgordo** [comment]
@bk201 this seems to be similar to:
https://github.com/harvester/harvester/issues/2651#issuecomment-1218280683 

Should we go ahead and close this out?

@mauriiciodias  -v1.2.0-rc2 is a "pre-release"- but since this does seem similar to 2651 then v1.2.0-rc2 resolves this issue.

**Comment by mauriiciodias** [comment]
@irishgordo Maybe we have the problem in another step.
Yes, I'm testing with the recommended version, where the iso was uploaded via IPMI on a Dell R640 server

The installation is frozen at this step:

<img width="744" alt="image" src="https://github.com/harvester/harvester/assets/91493872/a70122e0-4e51-40cd-b986-9088e6c090a8">

**Comment by bk201** [comment]
Duplicated with https://github.com/harvester/harvester/issues/2651, feel free to open again if v1.2.0 doesn't fix this. Thanks @irishgordo

**Comment by CalWooten95** [comment]
So I've been trouble shooting this issue the past few days, what I found to fix it was an error with no default route being provided to my machine via DHCP. Because of this I could see in the installer logs that the harvester kubernetes cluster was not being built because a default route is required for the CNI when installing kubernetes (even if the default route isn't functional).

Once I provided a default route through my dhcp it was able to finish the run.sh and start the next step of grabbing the kubernetes images. Woohoo!
